portal-api/src/v1/services/utilisation-report/email-service.ts

```diff
@@ -0,0 +1,82 @@
+import { array, string } from 'zod';
+import { PaymentOfficerTeam, asString } from '@ukef/dtfs2-common';
+import sendEmail from '../../email';
+import { EMAIL_TEMPLATE_IDS } from '../../../constants';
+import { formatDateForEmail } from '../../helpers/formatDateForEmail';
+import api from '../../api';
+import { InvalidEnvironmentVariableError } from '../../errors';
+
+const getUkefGefReportingEmailRecipients = () => {
+  const EmailsSchema = array(string());
+
+  try {
+    const emails =  EmailsSchema.parse(JSON.parse(asString(process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT, 'UKEF_GEF_REPORTING_EMAIL_RECIPIENT')));
```

Comment:
This requires emails to be provided as: 
`UKEF_GEF_REPORTING_EMAIL_RECIPIENT=["email1", "email2"]`
The env var is just read as a string so we can change this to a different form if we could like, just anything we can then parse and turn into an array